src/theme/QueryTable/QueryTable.js

```diff
@@ -40,6 +41,12 @@ export const QueryTable = ({
     </LoadNewButton>
   ) : null;
 
+  useEffect(() => {
+    if (isFetched && typeof props.onFetched === 'function') {
```

Comment:
`onFetched` seems to only be part of the `QueryTable` api, `Table` will never need it?

Maybe better to destruct it in `QueryTable` props to prevent passing dead props to Table.
```export const QueryTable = ({ queryHook, xxx, transformResponse,  onFetched, ...props```

Reply:
Total agree with you, but this would be refactoring QueryTable. I have no idea how many places I'll break. This deserves a dedicated ticket.

Reply:
I think you misunderstood me but then i see your last commit and it seems like you understood me after all :D